packages/simpleserialize.com/src/components/Input.tsx

```diff
@@ -33,7 +33,7 @@ type State = {
   userHasEditedInput: boolean;
 };
 
-const DEFAULT_FORK = "deneb";
+const DEFAULT_FORK = "deneb/index.js";
```

Comment:
@nazarhussain do you remember why this change was required? 

it breaks the website
```
index.tsx:17 TypeError: Cannot convert undefined or null to object
    at Object.keys (<anonymous>)
    at typeNames (types.ts:35:17)
    at Input.names (Input.tsx:101:12)
    at Input.render (Input.tsx:284:27)
```

I don't see any issues when reverting this to previous code, but might be missing something